internal/cmd/kafka/confluent_kafka_configs.go

```diff
@@ -72,6 +73,10 @@ func getConsumerConfigMap(group string, kafka *configv1.KafkaClusterConfig, clie
 	}
 	log.CliLogger.Debugf("Created consumer group: %s", group)
 
+	if err := configMap.SetKey("enable.auto.commit", false): err!=nil {
```

Comment:
Doesn't look like this code compiles. Has it been tested?
```suggestion
	if err := configMap.SetKey("enable.auto.commit", false); err != nil {
```

Reply:
thanks! fixed build and tested once again